partner_credit_limit/model/payment_term.py

```diff
@@ -0,0 +1,23 @@
+
+# -*- coding: utf-8 -*-
+############################################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 2011 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: hugo@vauxoo.com
+#    planned by: Nhomar Hernandez <nhomar@vauxoo.com>
+############################################################################
+from openerp import models, fields
+
+
+class PaymentTerm(models.Model):
+
+    _inherit = 'account.payment.term'
+    _order = 'sequence'
+
+    sequence = fields.Integer(
+        'Sequence', required=True,
+        default=lambda self: self.env['ir.sequence'].get(
```

Comment:
`sequence` is special field of Odoo why you define one default with `ir.sequence` model?

You can see in this field the behavior of sequence,
https://github.com/odoo/odoo/blob/8.0/openerp/addons/base/ir/ir_mail_server.py#L177